src/main/resources/application.yaml

```diff
@@ -87,7 +87,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment}
   api.url: ${idam_url:https://idam-api.aat.platform.hmcts.net}
 
 oidc.issuer: ${OIDC_ISSUER_URL:https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts}
```

Comment:
Any reason this variable `PRD_S2S_AUTHORISED_SERVICES`  hasn't set in helm charts values.yaml ?

Reply:
currently we are getting the s2s-authorised services  values from application.yml file itself .